src/mesh/impls/bout/boutmesh.hxx

```diff
@@ -263,6 +263,13 @@ private:
 
   int MXG, MYG, MZG; // Boundary sizes
 
+  // Grid file provenance tracking info
+  std::string grid_id = "";
+  std::string hypnotoad_version = "";
+  std::string hypnotoad_git_hash = "";
+  std::string hypnotoad_git_diff = "";
+  std::string hypnotoad_geqdsk_filename = "";
```

Comment:
Just to note that when we get the new I/O stuff in, we won't need to have these as class members